### PR TITLE
[ACID][Instance] shh - fix minor bugs found in latest tests

### PR DESCRIPTION
**Pull request by miraco**
add special ai handling for sleeping acolyte and slepping brawlers correct shadowmoon acolyte timers
remove shattered hand legionnaire aura of discipline fix some more CallForHelp based on wotlk classic pulls

add custom UnitFlags to Dummys to fix the bug where shattered hand gladiators and shattered hand centurion attack dummys due to different factions

### CODE REVIEW

File: Updates/0367_shattered_halls_fixes.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+-- Same for Heroic
+UPDATE creature_template SET CallForHelp = '6' WHERE entry IN (20590, 20567, 20594);
+
+
+-- Shattered Hand Legionnaire Aura of Discipline handled via sd2 script
+DELETE FROM creature_template_addon WHERE entry = 16700;
+
+-- Custom change to fix shattered halls dummys getting attacked by gladiators due to different factions
+-- Before: update creature_template set UnitFlags = '2304' where entry = 17578;
+UPDATE creature_template SET UnitFlags = UnitFlags|640 WHERE entry = 17578;
```
Comment (AnonXS) [inline Updates/0367_shattered_halls_fixes.sql:25]: cmangos_custom.sql? we dont know how or why etc.

Comment (miraco) [reply Updates/0367_shattered_halls_fixes.sql:25]: yes can split it into own file, and even tho its a custom change, its currently the best way to precent them fighting each other

Comment (AnonXS) [reply Updates/0367_shattered_halls_fixes.sql]: i know, we've had this npc as custom change for a long time, but we should keep it this way, incase we find the answer to npcs reacting how they are supposed to.

Comment (miraco) [reply Updates/0367_shattered_halls_fixes.sql]: So you want me to complete remove the custom change?

**Pull request merged**